code/datums/mind_vr.dm

```diff
@@ -0,0 +1,2 @@
+/datum/mind
+	var/vore_death = FALSE	// Was our last gasp a gurgle?
```

Comment:
This is good yes.

Reply:
There is room for bypassable abuse there still, but hopefully should it really happen, it will be spotted.